<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="c0016390" failures="17" tests="114" name="eraRerate.eraRerateTestNGSlow" time="3114.769" errors="9" timestamp="2021-04-23T21:32:00 UTC" skipped="23">
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="26.393"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.169"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="125.691">
    <failure message="Failed Selecting Contact Method and Clicking Continue" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1548)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.174"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.176"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.179"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.196"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="124.585">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.196"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.170"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.194"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="27.666"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.194"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="44.981">
    <failure message="Adjustment can not be done for discount amount" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Adjustment can not be done for discount amount
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1457)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="75.459">
    <failure message="After Rerate Click No Data Came -- Unknown Error" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: After Rerate Click No Data Came -- Unknown Error
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1307)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.196"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="48.081">
    <failure message="" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1457)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="40.854">
    <failure message="After Rerate Click No Data Came -- Unknown Error" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: After Rerate Click No Data Came -- Unknown Error
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1287)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="39.879">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.175"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.192"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="37.870">
    <failure message="After Rerate Click No Data Came -- Unknown Error" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: After Rerate Click No Data Came -- Unknown Error
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1287)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.170"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.195"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="39.227">
    <failure message="After Rerate Click No Data Came -- Unknown Error" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: After Rerate Click No Data Came -- Unknown Error
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1287)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod4" time="0.000">
    <error message="
Data Provider public synchronized java.lang.Object[][] eraRerate.eraRerateTestNGSlow.dataProviderMethod4() returned a null value" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] eraRerate.eraRerateTestNGSlow.dataProviderMethod4() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:147)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod4 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="97.600"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.170"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="31.901">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.198"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="54.143">
    <error message="Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;C0016390&#039;, ip: &#039;10.242.69.100&#039;, os.name: &#039;Windows Server 2016&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;11.0.6&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'C0016390', ip: '10.242.69.100', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1228)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="33.152">
    <failure message="Could Not Get To Charge Code Details" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Could Not Get To Charge Code Details
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:980)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="36.585">
    <failure message="After Rerate Click No Data Came -- Unknown Error" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: After Rerate Click No Data Came -- Unknown Error
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1287)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="34.167">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="41.634">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="127.572">
    <failure message="Failed Selecting Contact Method and Clicking Continue" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1548)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.203"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod2" time="0.000">
    <error message="
Data Provider public synchronized java.lang.Object[][] eraRerate.eraRerateTestNGSlow.dataProviderMethod2() returned a null value" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] eraRerate.eraRerateTestNGSlow.dataProviderMethod2() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:147)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod2 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.192"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.193"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="34.011">
    <failure message="Could Not Get To Charge Code Details" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Could Not Get To Charge Code Details
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:980)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.542">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.218"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.172"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.171"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="54.614">
    <error message="Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;C0016390&#039;, ip: &#039;10.242.69.100&#039;, os.name: &#039;Windows Server 2016&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;11.0.6&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'C0016390', ip: '10.242.69.100', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1228)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.181"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="41.659">
    <failure message="" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1457)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="26.841"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.171"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="39.302">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="29.112"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="54.431">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.201"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="78.370">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.195"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod3" time="0.000">
    <error message="
Data Provider public synchronized java.lang.Object[][] eraRerate.eraRerateTestNGSlow.dataProviderMethod3() returned a null value" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] eraRerate.eraRerateTestNGSlow.dataProviderMethod3() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:147)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod3 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.169"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.193"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="29.269"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.196"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.196"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.459">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="77.059">
    <failure message="After Rerate Click No Data Came -- Unknown Error" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: After Rerate Click No Data Came -- Unknown Error
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1307)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.170"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.203"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.173"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="40.693">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.606">
    <error message="Cannot locate element with text: Fedex 2Day Freight  (80)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;C0016390&#039;, ip: &#039;10.242.69.100&#039;, os.name: &#039;Windows Server 2016&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;11.0.6&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: Fedex 2Day Freight  (80)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'C0016390', ip: '10.242.69.100', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1218)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.196"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.195"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="55.094">
    <error message="Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;C0016390&#039;, ip: &#039;10.242.69.100&#039;, os.name: &#039;Windows Server 2016&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;11.0.6&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'C0016390', ip: '10.242.69.100', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1228)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.202"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.192"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.764">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.864">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="44.803">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="123.574">
    <failure message="Failed Selecting Contact Method and Clicking Continue" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1548)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.194"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="54.845">
    <error message="Cannot locate element with text: Fedex 1Day Freight  (70)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;C0016390&#039;, ip: &#039;10.242.69.100&#039;, os.name: &#039;Windows Server 2016&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;11.0.6&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: Fedex 1Day Freight  (70)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'C0016390', ip: '10.242.69.100', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1218)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="28.362"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="77.666">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="42.274">
    <failure message="Cannot Credit For More Than The Charge Amount Due." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Cannot Credit For More Than The Charge Amount Due.
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1457)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.198"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.798">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.168"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.195"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.174"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="36.520">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="28.163"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="16.200">
    <failure message="Failed on Entering Tracking Number" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed on Entering Tracking Number
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:885)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="31.192"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="54.773">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.216"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.192"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="26.108"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="27.642"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="26.061"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="29.467"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="27.931"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="122.420">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.195"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.195"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="46.306">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="34.385">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.194"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.192"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.202"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.206"/>
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="126.358">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="53.692">
    <error message="Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;C0016390&#039;, ip: &#039;10.242.69.100&#039;, os.name: &#039;Windows Server 2016&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;11.0.6&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text: 
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'C0016390', ip: '10.242.69.100', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:147)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:1228)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="42.604">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="33.877">
    <failure message="Could Not Get To Charge Code Details" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Could Not Get To Charge Code Details
at org.testng.Assert.fail(Assert.java:99)
at eraRerate.eraRerateTestNGSlow.doEraRerate(eraRerateTestNGSlow.java:980)
at eraRerate.eraRerateTestNGSlow.testMethod1(eraRerateTestNGSlow.java:574)
at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="eraRerate.eraRerateTestNGSlow" name="testMethod1" time="0.171"/>
  <system-out/>
</testsuite> <!-- eraRerate.eraRerateTestNGSlow -->
